### PR TITLE
Simplify claim steps

### PR DESCRIPTION
**Pull request by handshake-enthusiast**
Remove unnecessary hs-client installation (see https://github.com/handshake-org/hsd/issues/638#issuecomment-931253002).

### CODE REVIEW

File: claim/index.html [hunk 1]
```diff
@@ -100,27 +100,25 @@ <h2>System Setup</h2>
   <p>Make sure you have nodejs and npm installed first. On MacOS, please install <a href="https://brew.sh">homebrew</a> and run "<b>brew install node unbound</b>". On debian/ubuntu, you can run "<b>sudo apt-get install nodejs npm build-essential libunbound-dev</b>". If you run other distributions or OSes, you can probably figure this part out.</p>
   <p>Next, install node-gyp: <b>npm install node-gyp</b></p>
   <h2>Download</h2>
-  <p>Download <b>hsd</b>, <b>hs-client</b>, and <b>hs-airdrop</b> from <a href="https://handshake.org/download/">https://handshake.org/download/</a>. If downloaded from github, the directory structure is slightly different (hsd-2.4.0/hsd should be replaced with just hsd in these instructions).</p>
-  <p>Extract hsd, hs-client, and hs-airdrop: <b>tar xvf hs*</b></p>
+  <p>Download <b>hsd</b> and <b>hs-airdrop</b> from <a href="https://handshake.org/download/">https://handshake.org/download/</a>. If downloaded from github, the directory structure is slightly different (hsd-3.0.1/hsd should be replaced with just hsd in these instructions).</p>
```
Comment (pinheadmz) [inline claim/index.html]: I'd like to not have to update this html page every time we release a new version - can it be worded so "latest release" works instead? Or maybe just "for example..."

**Comment by pinheadmz** [comment]
Good idea, thanks. Let's just see if we can clean up the version numbers somehow. I think `hsd-*` was being used for future-proofing

File: claim/index.html (hunk 1 above)
Comment (handshake-enthusiast) [reply claim/index.html]: I just saw `hsd-2.4.0/hsd` above and decided to go ahead with the explicit versions, so it's easier to run (copy, paste, enter, success). OK, I can update a bit later. Alternatively, I can offer to update it every time when a new version is released.

Comment (handshake-enthusiast) [reply claim/index.html]: Updated to a wildcard, so you can proceed with merging as you prefer. My offer is still valid though 😉

**Pull request merged**